torch/_decomp/decompositions.py

```diff
@@ -301,8 +301,8 @@ def _prelu_kernel_backward(
 def rrelu_with_noise(
     self: Tensor,
     noise: Tensor,
-    lower: float,
-    upper: float,
+    lower: float = 1.0 / 8,
+    upper: float = 1.0 / 3,
```

Comment:
can you add a test somewhere?

Reply:
I added a test that is similar to the original bug report. Not sure if it is in the right place though.